widgetry/src/backend_glow_native.rs

```diff
@@ -67,6 +67,8 @@ pub fn setup(window_title: &str) -> (PrerenderInnards, winit::event_loop::EventL
         );
     }
 
+    load_textures(&gl, "system/assets/textures/spritesheet.png", 64).unwrap();
```

Comment:
Currently the sprite sheet are assets snipped from https://www.kenney.nl/assets/map-pack

Whose license is:

> License: (CC0 1.0 Universal) You're free to use these game assets in any project, personal or commercial. There's no need to ask permission before using these. Giving attribution is not required, but is greatly appreciated!

So if we do want to use these, it would be nice to add a little shoutout somewhere.